### PR TITLE
up test coverage of BinaryReader to 100%

### PR DESCRIPTION
**Pull request by ixje**
**What current issue(s) does this address?, or what feature is it adding?**
up `BinaryReader` coverage to 100%

**How did you solve this problem?**
write missing tests

**How did you make sure your solution works?**
`make test && make coverage`

**Did you add any tests?**
yes

**Are there any special changes in the code that we should be aware of?**
fixed unreachable code in `BinaryReader`

### CODE REVIEW

File: neocore/IO/BinaryReader.py [hunk 1]
```diff
@@ -223,8 +223,8 @@ def ReadVarInt(self, max=sys.maxsize):
             int:
         """
         fb = self.ReadByte()
-        if fb is None:
-            return 0
+        if fb is 0:
```
Comment (ixje) [inline neocore/IO/BinaryReader.py]: because `self.ReadByte()` never returns `None` but actually `0`.

Comment (metachris) [reply neocore/IO/BinaryReader.py]: `if not fb` might also work

**Comment by metachris** [comment]
Looks great 👍 

`make lint` fails ^^

**Comment by ixje** [comment]
ugh, didn't realise pycharm requires you to configure auto format for every project. I thought i had it set globally O_o

**Comment by metachris** [comment]
awesome, thanks

**Pull request merged**